contracts/kleros/KlerosLiquid.sol

```diff
@@ -458,9 +458,7 @@ contract KlerosLiquid is TokenController, Arbitrator {
         require(newNextDelayedSetStake >= nextDelayedSetStake);
         for (uint i = nextDelayedSetStake; i < newNextDelayedSetStake; i++) {
             DelayedSetStake storage delayedSetStake = delayedSetStakes[i];
-            this.call( // solium-disable-line security/no-low-level-calls
-                abi.encodeWithSignature("_setStake(address,uint96,uint128)", delayedSetStake.account, delayedSetStake.subcourtID, delayedSetStake.stake)
-            ); // Intentional use to avoid blocking.
+            _setStake(delayedSetStake.account, delayedSetStake.subcourtID, delayedSetStake.stake);
```

Comment:
Shouldn't we handle return value of `_setStake`. If false add a delayed action? @epiqueras

Reply:
See Slack.